src/api/event/content-types/event/lifecycles.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+
+    async afterCreate(event) {
+        
+        // Extract the ID of the created event
+        const { id, entityId, entityType } = event.result;
+
+        try {
+            // Fetch all users with active subscriptions for the given entityId and entityType
+            const subscriptions = await strapi.entityService.findMany('api::subscription.subscription', {
+                filters: {
+                    entityId: entityId,
+                    entityType: entityType,
+                    status: 'active',
+                },
+                populate: ['user'],
```

Comment:
Do we need to populate with user data? I think without populate, strapi will still return the user ID which is what we need to generate notification.

Reply:
FYI. Strapi doesn't include any data on the relation without populate parameter. This is done to correctly control published and allowed data. And that is probably the reason why even one-to-one relationship like for example user to profile are actually stored in many-to-many intermediary table. 
https://docs.strapi.io/dev-docs/api/rest/populate-select#population 
```The REST API by default does not populate any type of fields, so it will not populate relations, media fields, components, or dynamic zones unless you pass a populate parameter to populate various field types.```
One would think that for EntityService, they could have make it more straightforward, but from what I saw it behaves the same way, without `populate` parameter in the example above subscription.user will be undefined.